cremalab/crema-app-web
# Add test:types script to type check entire project

## Description

Adds a `test:types` script that will type check every `.ts` or `.tsx` file in the `src` directory. This ensures that unit and story files do not contain type errors in them.

Closes #43

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [x] `test:types` returns type errors in any file

### 👩‍🔬 Test Instructions

1. Pull down branch
1. Create a type error in a test file
1. Run `npm run test:types`
1. Ensure that error is caught

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


---

>### Roles & Responsibilities
>
>#### 👨‍💻 Assignee
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Reviewer
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
